### PR TITLE
[release/v2.1] update k8s versions

### PR DESCRIPTION
**Pull request by superseb**

### CODE REVIEW

File: apis/management.cattle.io/v3/k8s_defaults.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	DefaultK8s = "v1.13.5-rancher1-1"
+	DefaultK8s = "v1.13.9-rancher1-2"
```
Comment (deniseschannon) [inline apis/management.cattle.io/v3/k8s_defaults.go]: why isn't it v1.13.9-rancher1-1

File: apis/management.cattle.io/v3/k8s_defaults.go [hunk 2]
```diff
@@ -957,6 +957,32 @@ var (
 			IngressBackend:            m("k8s.gcr.io/defaultbackend:1.4"),
 			MetricsServer:             m("gcr.io/google_containers/metrics-server-amd64:v0.3.1"),
 		},
+		"v1.13.9-rancher1-2": {
```
Comment (deniseschannon) [inline apis/management.cattle.io/v3/k8s_defaults.go]: why isn't it v1.13.9-rancher1-1?

**Comment by deniseschannon** [comment]
LGTM

**Comment by cjellick** [comment]
LGTM

**Pull request merged**